### DISCUSSION TITLE
Failed to import pysu2 after built from source code

### DISCUSSION THREAD

**Discussion by zhang-qiang-github** [su2code/SU2#1583] [Q&A]
**Describe the bug**
I have successfully built the `SU2`. Then, I import the `pysu2`, and a bug is reported:

```
(SU2) zhq@ubuntu:~/Desktop/SU2$ python
Python 3.7.11 (default, Jul 27 2021, 14:32:16) 
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pysu2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/zhq/Desktop/SU2/bin/pysu2.py", line 20, in <module>
    _pysu2 = swig_import_helper()
  File "/home/zhq/Desktop/SU2/bin/pysu2.py", line 17, in swig_import_helper
    return importlib.import_module(mname)
  File "/home/zhq/.conda/envs/SU2/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ValueError: mpi4py.MPI.Status size changed, may indicate binary incompatibility. Expected 56 from C header, got 48 from PyObject
>>> 

```

**To Reproduce**
I just build the `SU2` from the source code as the recommanded:

```
python3 ./meson.py build -Denable-pywrapper=true --prefix=/home/zhq/Desktop/SU2/
./ninja -C build install
```
Then, enter the python environment, and import `pysu2`. 

**Desktop (please complete the following information):**
 - OS: Ubuntu 18
 - C++ compiler and version: 
     1. gcc version 7.5.0 (Ubuntu 7.5.0-3ubuntu1~18.04)
     2. gcc 7.5.0 "c++ (Ubuntu 7.5.0-3ubuntu1~18.04)
 - MPI implementation and version: mpi4py-3.1.3
 - SU2 Version: 7.3.0 "Blackbird"

Any suggestion is appreciated~~~

**Comment by pcarruscag** [comment]
They look like different versions of python to me.
You are calling python3 for one thing and python for the other.

**Comment by zhang-qiang-github** [comment]
Finally, I solve this problem by: `pip3 install mpi4py --no-binary mpi4py --user --force --no-cache-dir`
